core/templates/dev/head/pages/topic_editor/TopicEditorStateService.js

```diff
@@ -36,6 +36,7 @@ oppia.factory('TopicEditorStateService', [
     var _topic = TopicObjectFactory.createInterstitialTopic();
     var _topicRights = TopicRightsObjectFactory.createInterstitialRights();
     var _subtopicPages = [];
+    var _newSubtopicPages = [];
```

Comment:
Add single-line comments above this and the previous line, explaining what the arrays contain.

Reply:
Done.